components/drivers/spi/SConscript

```diff
@@ -28,7 +28,7 @@ if GetDepend('RT_USING_SFUD'):
     if rtconfig.CROSS_TOOL == 'gcc':
         LOCAL_CCFLAGS += ' -std=c99'
     elif rtconfig.CROSS_TOOL == 'keil':
-        LOCAL_CCFLAGS += ' --c99'
```

Comment:
Please keep this flag or could you please provide a better way for Keil MDK with ARMCC?

Reply:
The CM23 ARCH need to use mdk-armclang to build project. Current RT building script just only supports ARMCC, but armclang not.  I have no idea on rtconfig.CROSS_TOOL. I think I should rollback this modification and notice 'remove --c99' in board's README.md.